docs/fundamentals/database-collection.txt

```diff
@@ -0,0 +1,198 @@
+on the ``Flower`` model class to use the ``mongodb_alt`` connection:
+
+.. code-block:: php
+
+   class Flower extends Model
+   {
+       protected $connection = 'mongodb_alt';
+   }
+
+.. _laravel-access-coll:
+
+Access a Collection
+-------------------
+
+When you create a ``Model`` class in {+odm-short+}, it is automatically
+stored in a collection with a name that is the plural of the model class
+name. For example, if you create a model class called ``Flower``,
+Laravel applies the model to the ``flowers`` collection in the database.
+
+.. tip::
+
+   You might be storing data in a collection with a different name than the
+   model, or your collection might contain more than one type of data. To
+   learn how to set a collection name when defining a model class, see the
+   :ref:`laravel-model-customize-collection-name` section of the Eloquent
+   Model Class guide.
+
+For many operations, such as CRUD operations and relationships, we
+recommend that you use the Eloquent ORM to access a collection
+implicitly. The following example specifies a find operation by using
+the ``Flower`` class, so Laravel retrieves results from the ``flowers``
+collection:
+
+.. code-block:: php
+
+   Flower::where('name', 'Water Lily')->get()
+
+For other collection operations, you can access a collection by using
+a facade to access the query builder class ``DB`` and calling the
+``collection()`` method. The following example shows the same query as
+in the preceding example constructed by using the ``DB::collection()``
+method:
+
+.. code-block:: php
+
+   DB::connection('mongodb')
+       ->collection('flowers')
+       ->where('name', 'Water Lily')
+       ->get()
+
+.. tip::
+   
+   In MongoDB, each document in a collection has a field called ``_id`` which has a value
+   that is a unique identifier. To learn more about customizing this field
+   when you create a model to access a collection, see the :ref:`laravel-model-customize-primary-key` field of the Eloquent
+   Model Class guide.
+   
+List Collections
+----------------
+
+To see the names of the collections in a database, call the 
+``listCollections()`` method.
+
+The following example acceses a database connection, then
+calls the ``listCollections()`` method and outputs the collection names:
+
+.. code-block:: php
+
+   $collections = DB::connection('mongodb')->getMongoDB()->listCollections();
+   dump($collections);
+
+Drop a Collection
+-----------------
+
+We recommend performing a **migration** to handle dropping a collection.
+Deleting data from within a migration provides a controlled approach
+that ensures consistency, version control, and reversibility in your
+application.
+
+To learn more about migrations, see the
+:ref:`laravel-eloquent-migrations` section in the Schema Builder guide.
+
+To drop a collection idiomatically, the Laravel ``Schema`` facade provides the
+``drop()`` and ``dropIfExists()`` methods. The following example
```

Comment:
@GromNaN: Since dropping a nonexistent collection is a NOP in PHP, is there any practical difference between these two methods? Does it make sense to talk about both of them given the driver behavior?

Reply:
So the doc should tell both methods have the same effect and they exist for compatibility with the SQL Eloquent.

We could remove the collection existence check from this method?
https://github.com/mongodb/laravel-mongodb/blob/95c5b4fac69c2bd5ea01367e62d14dd5278e7513/src/Schema/Builder.php#L87-L92

Reply:
Correct. I see the `Blueprint::drop()` method calls `Collection::drop()`. That NOPs if the namespace doesn't exist.